tidepool_data_science_metrics/common/common.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+
+
+def mean(bg_values, round_val=2):
+    """
+    Calculate the mean within a set of glucose values
+
+    Parameters
+    ----------
+    bg_values : ndarray
+        1D array containing data with `int` type.
+    round_val : int
+        The number of digits to round the result to.
+
+    Returns
+    -------
+    int
+        The calculated Means
+    """
+    return round(np.mean(bg_values), round_val)
+
+
+def avg(bg_values, round_val=2):
+    """
+    Calculate the average within a set of glucose values
+
+    Parameters
+    ----------
+    bg_values : ndarray
+        1D array containing data with `int` type.
+    round_val : int
+        The number of digits to round the result to.
+
+    Returns
+    -------
+    int
+        The calculated Average
+    """
+
+    return round(np.average(bg_values), round_val)
```

Comment:
I had to look up why numpy had both mean and avg, and according to SO it's because average optionally accepts weights. Perhaps we can do the same here?

Reply:
yeah, that would be great